src/LeftNav/NavLink.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {PropTypes} from "react";
+import classnames from "classnames";
+
+import {FlexBox, FlexItem, ItemAlign} from "../";
+
+export function NavLink(props) {
+  const {cssClass} = NavLink;
+
+  const selected = props.selected ? cssClass.SELECTED : null;
+
+  return (
+    <FlexBox
+      component="li"
+      className={classnames(cssClass.CONTAINER, props.className, selected)}
+      onClick={props.onClick}
+      alignItems={ItemAlign.CENTER}
+    >
+      <FlexItem className={cssClass.ICON}>{props.icon}</FlexItem>
```

Comment:
fyi: no need for `FlexItem` unless you're actually using one of the flex properties provided